proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -203,12 +203,15 @@ message RecordActivityTaskStartedResponse {
 message RespondWorkflowTaskCompletedRequest {
     string namespace_id = 1;
     temporal.api.workflowservice.v1.RespondWorkflowTaskCompletedRequest complete_request = 2;
+    bool with_new_workflow_task = 3 [deprecated = true];
```

Comment:
what's the purpose of this field? and looks like it's value is always true.

Reply:
chatted offline... results on the history service side will dictate whether to execute the new-workflow-task logic